Tensile/Common.py

```diff
@@ -1618,6 +1618,8 @@ def assignGlobalParameters( config ):
     globalParameters["ROCmPath"] = os.environ.get("ROCM_PATH")
   if "TENSILE_ROCM_PATH" in os.environ:
     globalParameters["ROCmPath"] = os.environ.get("TENSILE_ROCM_PATH")
+  if "CMAKE_CXX_COMPILER" in os.environ:
+    globalParameters["CmakeCxxCompiler"] = os.environ.get("CMAKE_CXX_COMPILER")
```

Comment:
We shouldn't have globalParameters that only sometimes exist.  This causes issues with testing.  Using `None` as a default value would work.

Reply:
Fixed in latest commit